### PR TITLE
Added description in skills for knowledge category.

### PR DESCRIPTION
**Pull request by dynamitechetan**
Partially Fixes issue #234 
Added description in skill for knowledge category.

### CODE REVIEW

**Review by DravitLochan — APPROVED** [review]

**Review by madhavrathi — APPROVED** [review]

**Review by Manveer695 — APPROVED** [review]

File: models/general/knowledge/en/atomic.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Tell the atomic mass of * | atomic mass of * | * atomic mass| find atomic mass of *| do you know the atomic mass of * | do you know atomic mass of * | find the atomic mass of * | search atomic mass of * | tell me the atomic mass of * 
+!description:the mass of an atom of a chemical element expressed in atomic mass units. It is approximately equivalent to the number of protons and neutrons in the atom (the mass number) or to the average number allowing for the relative abundances of different isotopes.
```
Comment (saurabhjn76) [inline models/general/knowledge/en/atomic.txt]: make T capital

File: models/general/knowledge/en/describe.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Describes something
 describe *
+!description: skill to describe anything
```
Comment (saurabhjn76) [inline models/general/knowledge/en/describe.txt]: s capital

File: models/general/knowledge/en/wikipedia.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # searches wikipedia for a given query
 wiki *| wiki for *| search wiki for *|* wiki|give me wiki results for *| tell me what wiki has to say on *| search wikipedia for *|show me wikipedia results on *| find * on wiki|what wiki says on *
+!description: searches wikipedia for a given query
```
Comment (saurabhjn76) [inline models/general/knowledge/en/wikipedia.txt]: Capitalize the first letter

File: models/general/knowledge/en/github.txt [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 #returns name
 Whose github username is *|Who is * on github | Name of github user * | What is the name of * on Github | Name of * on Github | Real name of * on Github | Tell me name of the github user *
+!description: gives information about a username on github
```
Comment (saurabhjn76) [inline models/general/knowledge/en/github.txt]: Capitalize the first letter

File: models/general/knowledge/en/iceland.txt [hunk 5]
```diff
@@ -2,6 +2,7 @@
 
 #Ex: name any concert in iceland
 * concert in iceland
+!description: gives details about concerts in iceland
```
Comment (saurabhjn76) [inline models/general/knowledge/en/iceland.txt]: Capitalize the first letter

File: models/general/knowledge/en/gender.txt [hunk 6]
```diff
@@ -1,6 +1,7 @@
 #returns gender of a person from his/her first name
 
 I am *|My name is *
+!description: returns gender of a person from his/her first name
```
Comment (saurabhjn76) [inline models/general/knowledge/en/gender.txt]: Capitalize the first letter

File: models/general/knowledge/en/domain_check.txt [hunk 7]
```diff
@@ -1,6 +1,7 @@
 #checks if a domain is a valid one for a mailing address or emailID
 
 validate email domain *|check email domain *|status of the email domain *
+!description: checks if a domain is a valid one for a mailing address or emailID
```
Comment (saurabhjn76) [inline models/general/knowledge/en/domain_check.txt]: Capitalize the first letter

File: models/general/knowledge/en/dictionary.txt [hunk 8]
```diff
@@ -1,5 +1,6 @@
 #words with a meaning similar to tree
 define *| Meaning of *| one word for *
+!description: gives words with a meaning similar to another word in spelling, in sound, in rhyme etc.
```
Comment (saurabhjn76) [inline models/general/knowledge/en/dictionary.txt]: Capitalize the first letter

**Review by saurabhjn76 — COMMENTED** [review]
Great work:+1: little changes just make the first letter capitalize at some places.

**Review by saurabhjn76 — APPROVED** [review]

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by dynamitechetan** [comment]
@mariobehling  resolved the conflicts.

**Pull request merged**